templates/artist/artist_detail.html

```diff
@@ -22,7 +22,7 @@ <h1 class="header-font">{{ artist.name }}</h1>
     <div class="break">
         <hr>
     </div>
-    <img src="{{ artist.photo.img.url }}" />
+    {% include "artist/includes/artist_detail.html" with artist=artist %}
```

Comment:
This thumbnail isn't used on multiple pages, so the `{% include %}` tag isn't really necessary here. You can just put the `{% thumbnail %}...{% endthumbnail %}` portion here instead and then get rid of `templates/artist/includes/artist_detail.html`. Don't forget to add `{% load thumbnail %}` to `templates/artist/artist_detail.html`!